packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -968,7 +969,7 @@ class NotifyingLogger extends Logger {
     if (!verbose) {
       return;
     }
-    _sendMessage(LogMessage('trace', message));
+    parent.printError(message);
```

Comment:
@annagrin this change is sufficient to get verbose logs in the terminal of VS Code

Reply:
This works, thanks! LGTM